sys/parsoid.js

```diff
@@ -344,7 +392,9 @@ class ParsoidService {
                         body: res.body[format].body
                     };
                     resp.headers.etag = mwUtil.makeETag(rp.revision, tid);
-                    return this.saveParsoidResult(hyper, req, format, tid, res)
+                    return this.saveParsoidResultToLatest(hyper, req, tid, res)
+                    .catch({ status: 412 }, () =>
```

Comment:
Hm... So the standard `If-Match/If-None-Match` doesn't fit because of the non-strict matching... Custom header is a possible solution indeed, but I'm not sure what do we gain from it?

Other response status candidate is:

> **409 Conflict**
Indicates that the request could not be processed because of conflict in the request, such as an edit conflict between multiple simultaneous updates.